converter.js

```diff
@@ -0,0 +1,69 @@
+// Get a reference to the button element in the DOM
+var button = document.getElementById("converter");
+button.addEventListener("click", determineConverter);
+document.querySelector("#input-text").addEventListener("keyPress",
```

Comment:
Here, instead of "#input-text" being inside your querySelector, instead add the same id you will add to your body html tag.